splitgpg2/__init__.py

```diff
@@ -117,6 +117,8 @@ def __init__(self, client_domain: str):
         # configuration options:
         self.verbose_notifications = False
         self.timer_delay = self.default_timer_delay()
+        #: allow client to provide passphrase
+        self.allow_client_passphrase = False
```

Comment:
Adding this comment here since I didn't find a buttion to add it to the commit:

Do we really want to allow passphrases from the client domain? This sounds like using the split-gpg2 wrongly.

In general I would rather consider dropping GENKEY completely (but that's somewhat orthogonal).

Reply:
It is necessary if one want GENKEY with the current gpg2 frontend. Even though the protocol supports `--no-protection` option, documentation advises to use `--passphrase ''` to set empty passphrase. I think it may be still possible to emit `--no-protection` option with `%no-protection` in `gpg2 --gen-key --batch`, I'll check that.

Reply:
Yes, `%no-protection` in batch mode use `GENKEY --no-protection` command. What do you think, should we allow `GENKEY --no-protection` by default? Or maybe only with some option set?
BTW Just filtering out those options triggers interactive pinentry (in the backend domain obviously). So, IMO it's fine to do that. Not sure how password caching interacts with it yet.

Reply:
@HW42 ping. I think I have addressed all the other comments, only waiting for this one to push the updated version.

Reply:
I would prefer to disable `GENKEY` by default.